tests/src/kakarot/test_kakarot.py

```diff
@@ -216,7 +224,7 @@ async def test_case(
         async def test_failing_contract(self, cairo_run):
             initial_state = {
                 CONTRACT_ADDRESS: {
-                    "code": bytes.fromhex("0xADD_CODE"),
+                    "code": bytes.fromhex("ADDC0DE1"),
```

Comment:
can fix the `0xADD_DATA` then as well

tbh this skipped test as been added as a placeholder for debugging a future contract, I think that we could even delete it

Reply:
We can keep the placeholder, but I changed this to show the actual format expected in this bytes.fromhex()